src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -931,4 +890,20 @@ public static MatchResults calculateMatchResults(final Fingerprint observedFp, f
     public void setReferenceFasta(final File referenceFasta) {
         this.referenceFasta = referenceFasta;
     }
+
+    public int getLocusMaxReads() {
+        return locusMaxReads;
+    }
+
+    public void setLocusMaxReads(final int locusMaxReads) {
+        this.locusMaxReads = locusMaxReads;
+    }
+
+    public String getDefaultSampleID() {
+        return defaultSampleID;
+    }
+
+    public void setDefaultSampleID(final String defaultSampleID) {
```

Comment:
This was added because there is a need to change the default sample ID outside of picard? It's not currently used.

Reply:
picard/fingerprint/ExtractFingerprint.java:102 

the getter isn't used, but I thought that if I'm using a setter I should probably have a getter....happy to remove it.

Reply:
It's OK